src/model_ode.cpp

```diff
@@ -394,6 +394,42 @@ void Model_ODE::fqBdot(realtype t, N_Vector x, N_Vector xB, N_Vector qBdot) {
     }
 }
 
+void Model_ODE::fxBdot_ss(const realtype t, const AmiVector &xB,
```

Comment:
If anyone has a better idea for the function name than just appending `_ss`, feel free to suggest... ;)